### ISSUE TITLE
A11y statement's: when not found, the a11y rating should be 1

### ISSUE THREAD

**Issue by marcusosterberg** [Webperf-se/webperf_core#927]
### URL

https://www.krokom.se

### Webperf_core version

2025.2.0

### What environment are you running?

- [ ] Using GitHub Actions
- [ ] Using Docker Image from Dock Hub
- [x] Local machine
- [ ] Other, please describe your environment in How to reproduce.

### Has anyone else reported it already?

- [x] No, there are no other issues related to this in [Issues](https://github.com/Webperf-se/webperf_core/issues).

### Expected Behavior

The overall rating should be the same as the a11y rating in this case. Since the only thing test 26 evaluates is the accessibility, and when the overall is 1 / 5 the a11y rating should also be 1.

### Actual Behavior.

Only the overall rating is set, not the a11y rating.

### Rating: 
- Overall: 1.0

### How to reproduce

```shell
python default.py -t 26 -L en -u https://www.krokom.se -r
```

### Log output

```shell
# Testing website https://www.krokom.se

## Test: 26 - Accessibility Statement

Started: 2025-02-03 14:14:31
Finished: 2025-02-03 14:19:37

### Rating: 
- Overall: 1.0

### Review:
 
#### Overall:
- Tested website: https://www.krokom.se
- No accessibility statement found ( 1.00 rating )
```

**Comment by 7h3Rabbit** [comment]
@marcusosterberg please see #926 for discussion on if this test only rates a11y or not.
This issue stand or breaks depending on the discussion.

**Comment by 7h3Rabbit** [comment]
![Image](https://github.com/user-attachments/assets/71ae1b34-71f7-4e6d-967b-78454283224f)
